### DISCUSSION TITLE
v1.6

### DISCUSSION THREAD

**Discussion by JonathanTreffler** [JonathanTreffler/backblaze-personal-wine-container#97] [Announcements]
### Added
- Added backblaze client auto-update functionality to the docker (#88, thanks @traktuner)

### Changed
- By default a known-good version of the backblaze client will now be used
  - Can be overridden by adding the environment variable "FORCE_LATEST_UPDATE=true"
- The wine version in the Dockerfiles is now pinned to get more control over stability

<hr /><em>This discussion was created from the release <a href='https://github.com/JonathanTreffler/backblaze-personal-wine-container/releases/tag/v1.6'>v1.6</a>.</em>

**Comment by seanwitzke** [comment]
<img width="1567" alt="Screenshot 2024-01-31 at 1 43 52 AM" src="https://github.com/JonathanTreffler/backblaze-personal-wine-container/assets/96857861/ce18041f-d5da-40e0-9f23-99ac83bb8d74">

Hey Jonathan, I keep running into this error in my docker and I dont know how to fix it. Any insight? Running the latest version of Unraid. I've tried changing folder permissions to nobody, root, my own user, etc. Still get this permissions denied error when its trying to make the wine container from the .exe @JonathanTreffler

**Reply by JonathanTreffler** [reply]
It is not just a matter of which user the config folder belongs to, but it needs to fit the ENV Variables USER_ID and GROUP_ID of the docker.
So if you set the owner to be nobody (which ist best on Unraid) you also need to set ENV vars accordingly, you can find out the UID and GID using

`id nobody`

should be 100 and 100, but I don't have an Unraid system to find out for sure.

**Reply by seanwitzke** [reply]
So:
docker run \
    -p 8080:5800 \
	-e USER_ID=100 \
	-e GROUP_ID=100 \
    --init \
    --name backblaze_personal_backup \
    -v "/mnt/user/:/drive_d/" \
    -v "/mnt/user/appdata/Backblaze_Personal_Backup/:/config/" \
    tessypowder/backblaze-personal-wine:latest

**Reply by JonathanTreffler** [reply]
Yes

**Comment by seanwitzke** [comment]
I will give it a whirl, and report back. I've been spending a few bunch of hours at this and it's been racking my brain.Thanks for helping me out.SeanOn Jan 31, 2024, at 5:05 PM, Jonathan Treffler ***@***.***> wrote:﻿
Yes
